project/Dependencies.scala

```diff
@@ -30,10 +30,10 @@ object Dependencies {
     "com.amazonaws" % "aws-java-sdk-kinesis" % awsVersion,
     "com.gu" %% "atom-manager-play" % atomLibVersion,
     "com.gu" %% "atom-publisher-lib" % atomLibVersion,
-    "com.gu" %% "editorial-permissions-client" % "2.0",
+    "com.gu" %% "editorial-permissions-client" % "2.15",
     "com.gu" %% "simple-configuration-ssm" % "1.5.6",
-    "com.gu" %% "fezziwig" % "1.2",
-    "com.gu" %% "pan-domain-auth-play_2-8" % "1.2.0",
+    "com.gu" %% "fezziwig" % "1.6",
+    "com.gu" %% "pan-domain-auth-play_2-9" % "3.0.1",
```

Comment:
I initially forgot to update this, but crucially, as we're updating to Play 2.9, we need to use `pan-domain-auth-play_2-9`!

Otherwise we get a `NoSuchMethodError` in the logs:

> Jan 26, 2024 @ 16:44:08.273	java.lang.NoSuchMethodError: 'void play.api.mvc.DiscardingCookie.<init>(java.lang.String, java.lang.String, scala.Option, boolean)'
> Jan 26, 2024 @ 16:44:08.273		at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
> Jan 26, 2024 @ 16:44:08.273		at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656
> Jan 26, 2024 @ 16:44:08.271		at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
> Jan 26, 2024 @ 16:44:08.271		at com.gu.pandomainauth.action.AuthActions.com$gu$pandomainauth$action$AuthActions$$discardCookies$(Actions.scala:102)